src/main/webapp/app/programming/shared/programming-exercise-update-timeline/programming-exercise-timeline.component.html

```
@@ -0,0 +1,198 @@
+<div class="frame">
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)